compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/v2/Scrollbar.desktop.kt

```diff
@@ -435,23 +438,11 @@ internal class SliderAdapter(
             return if (extraContentSpace == 0.0) 1.0 else extraScrollbarSpace / extraContentSpace
         }
 
-    private var rawPosition: Double
+    private val rawPosition: Double
         get() = scrollScale * adapter.scrollOffset
-        set(value) {
-            runBlocking {
-                adapter.scrollTo(value / scrollScale)
-            }
-        }
 
-    var position: Double
+    val position: Double
         get() = if (reverseLayout) trackSize - thumbSize - rawPosition else rawPosition
-        set(value) {
-            rawPosition = if (reverseLayout) {
-                trackSize - thumbSize - value
-            } else {
-                value
-            }
-        }
```

Comment:
I would still keep it in a function (`setPosition`). `setRawPosition` could be inlined inside it.

Reply:
Do you mean to keep the setter for `position`?

> setRawPosition could be inlined inside it.
But we can't call/inline a suspend fun setRawPosition from it?

Or maybe you meant something else?

Reply:
I meant that we should have 
```
suspend fun setPosition(position) {
   // Transform the position into raw position and call adapter.scrollTo
}
```